packages/nodes-base/nodes/Set/test/v2/raw.test.ts

```diff
@@ -0,0 +1,145 @@
+import { get } from 'lodash';
+import { constructExecutionMetaData } from 'n8n-core';
+import {
+	NodeOperationError,
+	type IDataObject,
+	type IExecuteFunctions,
+	type IGetNodeParameterOptions,
+	type INode,
+} from 'n8n-workflow';
+
+import { type SetNodeOptions } from '../../v2/helpers/interfaces';
+import * as utils from '../../v2/helpers/utils';
+import { execute } from '../../v2/raw.mode';
+
+export const node: INode = {
```

Comment:
Since we don't use this anywhere else let's remove export
```suggestion
const node: INode = {
```